ParseLiveQuery/src/test/java/com/parse/TestParseLiveQueryClient.java

```diff
@@ -498,6 +539,30 @@ private static JSONObject createObjectDeleteMessage(int requestId, ParseObject p
         return jsonObject;
     }
 
+    private static class LoggingCallbacks implements ParseLiveQueryClientCallbacks {
```

Comment:
I was more worried that the use of transcript could be misused if the class isn't instantiated but it looks to be declared private so can't be accessed by other tests.